docs/develop/porting.rst

```diff
@@ -155,6 +155,9 @@ The following is an example of an ``mpconfigport.h`` file:
    #define MICROPY_PY_IO                           (0)
    #define MICROPY_PY_STRUCT                       (0)
 
+   // fixes sys/usys import issue
```

Comment:
What issue did you run into exactly?  IMO it would be better to comment something like "Enable modules to be imported with their standard name, like sys"